### PR TITLE
Configurations for BRC Analytics

### PR DESCRIPTION
**Pull request by hujambo-dunia**

### CODE REVIEW

**Review by wm75 — COMMENTED** [review]
Cool, guess this can be merged once the `org_id` for `linkedin-brc` is updated.

**Comment by wm75** [comment]
@hujambo-dunia @scottcain what about enabling mastodon-brc (see above)?

File: plugins.yml [hunk 1]
```diff
@@ -53,3 +53,28 @@ plugins:
     enabled: true
     config:
       save_path: "markdown_cache"
+
+  - name: mastodon-brc
+    class: mastodon.mastodon_client
+    enabled: false
```
[the remark this answers is not in the document]
Comment (scottcain) [reply plugins.yml:59]: I think enabled is fine (though it certainly shouldn't be default)

[the remark this answers is not in the document]
Comment (wm75) [reply plugins.yml:59]: Where content gets posted to is determined exclusively by the media section in the header of your post.
The only default kind of behavior we have is that when you click "Create new post" on the repo's main page, we're prefilling the header, but that's independent of the config here.

**Comment by hujambo-dunia** [comment]
@scottcain enabled.

@wm75 thank you and let me know if you have further suggestions.

**Pull request merged**